code/controllers/subsystem/tts.dm

```diff
@@ -146,19 +151,22 @@ SUBSYSTEM_DEF(tts)
 		var/datum/tts_request/current_request = processing_messages[processing_messages.len]
 		processing_messages.len--
 		var/datum/http_request/request = current_request.request
-		if(!request.is_complete())
+		var/datum/http_request/request_blips = current_request.request_blips
+		if(!request.is_complete() || !request_blips.is_complete())
 			continue
 
 		var/datum/http_response/response = request.into_response()
+		var/datum/http_response/response_blips = request_blips.into_response()
 		in_process_http_messages -= current_request
 		average_tts_messages_time = MC_AVERAGE(average_tts_messages_time, world.time - current_request.start_time)
 		var/identifier = current_request.identifier
-		if(response.errored)
+		if(response.errored || response_blips.errored)
```

Comment:
Don't really like this implementation since it means if one HTTP requests fails, the other doesn't get to play.
Also for servers who want to use TTS with a weaker GPU, they may not want the double processing/they may want to force all their users onto TTS only.
Can you make this a config option instead?

Reply:
No.